client/js/screens/SignUp/styles.js

```diff
@@ -0,0 +1,50 @@
+import {StyleSheet} from 'react-native';
+import {THEME} from '../../config';
+
+const styles = StyleSheet.create({
+  profileWrap: {
+    alignItems: 'center',
+    marginTop: THEME.spacing.default(),
+    marginBottom: THEME.spacing.default(0.25),
+  },
+  profileImg: {
+    width: 169,
+    height: 169,
```

Comment:
use resizeMode with height and width %s.

Reply:
ok~

Reply:
If I set the height to %, it will expand a lot, because the screen is scrollable and it's really long.

![image](https://user-images.githubusercontent.com/20639360/70561119-be215800-1b3e-11ea-9352-5c26fcc0d012.png)

Reply:
Fixed